packages/rocketchat-lib/settings/server/startup.coffee

```diff
@@ -38,6 +38,9 @@ RocketChat.settings.addGroup 'General'
 RocketChat.settings.add 'Site_Name', 'Rocket.Chat', { type: 'string', group: 'General', public: true }
 RocketChat.settings.add 'Allow_Invalid_SelfSigned_Certs', false, { type: 'boolean', group: 'General' }
 RocketChat.settings.add 'Disable_Favorite_Rooms', false, { type: 'boolean', group: 'General' }
+RocketChat.settings.add 'Allowed_Domains', 'gmail.com;hotmail.com', { type: 'string', group: 'General'}
+RocketChat.settings.add 'At_Sign', '@', { type: 'string', group: 'General'}
+RocketChat.settings.add 'Values_Separator', ';', { type: 'string', group: 'General'}
```

Comment:
@rodrigok We can change this later, after defining the official domain´s whitelist  =)